Example/MatrioskaExample/MatrioskaCodeViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  MatrioskaCodeViewController.swift
+//  MatrioskaExample
+//
+//  Created by Mathias Aichinger on 19/01/2017.
+//  Copyright © 2017 runtastic. All rights reserved.
+//
+
+import UIKit
+import Matrioska
+import SnapKit
+
+class MatrioskaCodeViewController: UIViewController {
+
+    let rootComponent = ClusterLayout.stack(
+        children: [
+            tileComponent(meta: TileConfig(text: "One", color: .red)),
+            tileComponent(meta: TileConfig(text: "Two", color: .green)),
+            ClusterLayout.stack(
+                children: [
+                    tileComponent(meta: TileConfig(text: "A", color: .red)),
+                    tileComponent(meta: TileConfig(text: "B", color: .green)),
+                    tileComponent(meta: TileConfig(text: "C", color: .orange)),
+                    tileComponent(meta: TileConfig(text: "D", color: .yellow))
+                    ],
+                meta: ClusterLayout.StackConfig(axis: .horizontal)
+            ),
+            tileComponent(meta: TileConfig(text: "Three", color: .orange)),
+            tileComponent(meta: TileConfig(text: "Four", color: .yellow))
+            ],
+        meta: ClusterLayout.StackConfig(title: "Test",
+                                        spacing: CGFloat(10.0),
+                                        axis: .vertical,
+                                        preserveParentWidth: true,
+                                        backgroundColor: .blue)
+    )
```

Comment:
Why not returning this component when declaring the `matrioskaCodeBuilder` on AppDelegate?